example.md

```diff
@@ -162,6 +187,33 @@ Page number is not shown in current page, but it's shown on later pages.
 
 ---
 
+#### Slide background Images
+
+You can set an image background for a slide
```

Comment:
In fact, the background image syntax `![bg]()` has many powerful features like Deckset.

```markdown
## Common use

![bg](images/bg1.jpg)

---

## Disable filter

![bg original](images/bg1.jpg)

---

## Multiple backgrounds

These will align to horizontal.

![bg](images/bg1.jpg)
![bg](images/bg2.jpg)
![bg](images/bg2.jpg)

---

## Sizing

![bg 50%](images/bg1.jpg)
```

In my opinion, I want to include these example for use. On the other hand, example.md would have too many examples..